explainaboard/metrics/metric.py

```diff
@@ -366,7 +364,7 @@ def calc_confidence_interval(
             )
         # Do bootstrapping otherwise
         else:
-            n_elems = max(int(prop_samples * len(stats)), 1)
+            n_elems = len(stats)
```

Comment:
I think the name of this variable should be `sample_size`.
And it can be used too in the next line.

Reply:
Done. Rename `n_samples` to `num_iterations` to avoid confusion with `sample_size`.